src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img images">
+        <img
+           class="images__img"
+           src="images/imac.jpeg"
+           alt="monobloc apple"
+          >
+      </div>
```

Comment:
Думаю, лучше такое наименование классов сделать
```suggestion
      <div class="card__image image">
        <img
           class="image__imac"
           src="images/imac.jpeg"
           alt="monobloc apple"
          >
      </div>
```

Reply:
class="image__imac" может тут так и оставить img ведь на разных картах разные картинки могут быть может так случится что на каритнке будет телик samsung а стиль image__imac будто дескриминация)